src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayCompilerInternalOptions.cs

```diff
@@ -53,31 +53,26 @@ internal enum SymbolDisplayCompilerInternalOptions
         /// </summary>
         ReverseArrayRankSpecifiers = 1 << 5,
 
-        /// <summary>
-        /// Display `System.ValueTuple` instead of tuple syntax `(...)`.
-        /// </summary>
-        UseValueTuple = 1 << 6,
-
         /// <summary>
         /// Display `System.[U]IntPtr` instead of `n[u]int`.
         /// </summary>
-        UseNativeIntegerUnderlyingType = 1 << 7,
+        UseNativeIntegerUnderlyingType = 1 << 6,
```

Comment:
i changed these values to keep things contiguous (they're internal, so no breaking change here).  can keep them the same though if that is preferred.